pkg/containerimagebuilderpusher/kaniko.go

```diff
@@ -78,7 +79,7 @@ func (k *Kaniko) BuildAndPushContainerImage(buildOptions *BuildOptions, namespac
 
 	// create job
 	k.logger.DebugWith("Creating job", "namespace", namespace, "jobSpec", jobSpec)
-	job, err := k.kubeClientSet.BatchV1().Jobs(namespace).Create(jobSpec)
+	job, err := k.kubeClientSet.BatchV1().Jobs(namespace).Create(context.Background(), jobSpec, metav1.CreateOptions{})
```

Comment:
that requires a follow up PR for all those added `ctx` objects. these needs to be injected by callees